core/src/test/java/eu/fasten/core/data/metadatadb/MetadataDaoTest.java

```diff
@@ -342,21 +374,23 @@ public void insertDependencyTest() {
         long packageId = 8;
         long dependencyId = 42;
         var versionRange = new String[]{"1.0.0-1.9.9"};
-        var insertValues = Mockito.mock(InsertValuesStep3.class);
+        var insertValues = Mockito.mock(InsertValuesStep4.class);
```

Comment:
Is this a good class name? I am not sure what "step 4" refers to, but it seems that this is going to change again, every time there will be another step.

Reply:
@MihhailSokolov

Reply:
I didn't choose that name unfortunately :)
It's a class from jOOQ and `4` refers to the number of attributes that will be inserted in the database.
The number of attributes increased from 3 to 4 because we added dependency metadata.
So we can't really do anything about this naming.